packages/internal-lib-build/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "internal-lib-build",
-  "version": "4.0.0-dev",
+  "version": "4.0.0-extensibility-preview.0",
```

Comment:
This one falls into the same category as the comment below about the commerce-sdk-react not changing and being independent.. so do we have to post fix it with extensibility? I think the easy answer is just to do it, but this lib hasn't received any extensibility modifications.. so it's worth noting.